AFG3XXXSup/devAFG3XXXchannel.db

```diff
@@ -74,6 +74,30 @@ record(mbbo, "$(P)OUTPUT$(C):FUNC:SP")
 
 alias("$(P)OUTPUT$(C):FUNC", "$(P)OUTPUT$(C):FUNC:SP:RBV")
 
+record(ai, "$(P)OUTPUT$(C):RAMP:SYMMETRY")
+{
+    field(DTYP, "stream")
+    field(INP,  "@devAFG3XXX.proto getRampSymmetry($(C)) $(PORT)")
+    field(SCAN, "$(SCAN=5 second)")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:OUTPUT$(C):RAMP:SYMMETRY")
+    field(SDIS, "$(P)DISABLE")
+    field(EGU, "%")
+    field(PINI, "YES")
+}
+
+record(stringout, "$(P)OUTPUT$(C):RAMP:SYMMETRY:SP")
```

Comment:
Can we add a comment explaining why we're using `stringout` here if it's intentional? From a first pass it looks wrong, I see the record below has:

> #Stringout because can send either <value>, INF, MIN or MAX

If that also applies to this one can we add that comment here too.

Reply:
Judging by the manual (and similar records), when writing you can set a percentage, MIN (0) or MAX (100), but reading back it's always just the percentage.